mlrun/utils/helpers.py

```diff
@@ -1718,6 +1718,75 @@ def validate_single_def_handler(function_kind: str, code: str):
             )
 
 
+def validate_node_selectors(
```

Comment:
The function will never return False, so why is it returning a value at all?

Reply:
I saw other validation functions we are using returns a Boolean, so wanted to keep the same behavior

Reply:
But you never actually return False

Reply:
No, it is not. It will only work if you change the code to:
```
            if not pattern.match(item):
                if not handle_invalid(error_message):
                     return False
```
then the function `validate_node_selectors` will return False

Reply:
why? the return is happening after warning, it's not exiting the function.

Reply:
Added a comment to explain :)